src/WordTranslator.ts

```diff
@@ -61,14 +77,26 @@ const datewordTranslations = {
         [DATE_TODAY]: "vandaag",
         [DATE_LASTWEEK]: "deze week",
         [DATE_LASTMONTH]: "deze maand",
-        [DATE_LAST6MONTH]:"deze 6 maanden",
+        [DATE_LAST6MONTH]: "deze 6 maanden",
         [DATE_LASTYEAR]: "dit jaar",
         [DATE_RANGE_PICK]: "kies een datum bereik...",
         [DATE_BETWEEN]: "Tussen...",
     },
+    [SPANISH]: {
+        [DATE_ON]: "en...",
+        [DATE_FROM]: "Desde...",
+        [DATE_UNTIL]: "Hasta...",
+        [DATE_TODAY]: "Hoy",
+        [DATE_LASTWEEK]: "La semana pasada",
+        [DATE_LASTMONTH]: "El mes pasado",
+        [DATE_LAST6MONTH]: "Ultimos 6 meses",
```

Comment:
I don't know if it matters, but capitalization of these strings differs from the other translations.